### ISSUE TITLE
navigation-emitter: ignore disabled elements

### ISSUE THREAD

**Issue by ianmcburnie** [makeup/makeup-js#87]
To support https://github.com/eBay/ebayui-core/issues/1757, any item that has `aria-disabled=true` should be filtered from list.

https://github.com/makeup/makeup-js/blob/master/packages/makeup-navigation-emitter/src/index.js#L127
